public/app/percona/backup/components/BackupInventory/RestoreBackupModal/RestoreBackupModal.tsx

```diff
@@ -38,12 +42,111 @@ export const RestoreBackupModal: FC<RestoreBackupModalProps> = ({
 }) => {
   const styles = useStyles(getStyles);
   const initialValues = useMemo(() => (backup ? toFormProps(backup) : undefined), [backup]);
+
+  const [selectedTimerange, setSelectedTimerange] = useState<Timeranges>();
+  const [selectedTimerangeFromDatepicker, setSelectedTimerangeFromDatepicker] = useState<DateTime>();
+  const [selectedDay, setSelectedDay] = useState<Date | undefined>();
   const handleSubmit = ({ serviceType, service }: RestoreBackupFormProps) => {
-    if (backup) {
+    if (backup && selectedTimerangeFromDatepicker) {
```

Comment:
what happens if I try to restore a non-PITR backup?

Reply:
Good catch 👍 Added solution for non-PITR backups